src/pages/Error/error.less

```diff
@@ -118,7 +118,7 @@
 }
 .error_main__500 .main_wrapper {
   background: transparent url('/server-353x310.png') no-repeat right top;
-  background-size: contain;
+  background-size: max(310px) auto;
```

Comment:
Before this, my longer error messages would cause the server to attack the text. ⚔️

![Screenshot 2024-01-31 at 4 12 04 AM](https://github.com/cfpb/sbl-frontend/assets/19983248/d157d844-9241-461a-aba9-194ac851bb1c)